tests/acceptance/features/apiSpaces/removeSpaces.feature

```diff
@@ -0,0 +1,181 @@
+@api @skipOnOcV10
+Feature: Remove files, folder and spaces
+  As a user
+  I want to be able to remove files, folders and space itself
+  Users with the editor role can also remove objects
+  Users with the viewer role cannot remove objects 
+
+  Note - this feature is run in CI with ACCOUNTS_HASH_DIFFICULTY set to the default for production
+  See https://github.com/owncloud/ocis/issues/1542 and https://github.com/owncloud/ocis/pull/839
+
+  Background:
+    Given user "Alice" has been created with default attributes and without skeleton files
+    And user "Brian" has been created with default attributes and without skeleton files
+    And the administrator has given "Alice" the role "Admin" using the settings api
+
+#   owner of space
```

Comment:
what is this line about?

Reply:
I separated tests by permissions: owner, editor, viewer